src/main/groovy/com/google/protobuf/gradle/ArchiveActionFacade.java

```diff
@@ -0,0 +1,50 @@
+package com.google.protobuf.gradle;
```

Comment:
Please add a file header for this new file.

Reply:
Oops, fixed :)